doc/modules/tree.rst

```diff
@@ -572,6 +572,35 @@ Mean Absolute Error:
 
 Note that it fits much slower than the MSE criterion.
 
+.. _tree_missing_value_support:
+
+Missing Values Support
+======================
+
+:class:`~tree.DecisionTreeClassifier` and :class:`~tree.DecisionTreeRegressor`
+have built-in support for missing values when `splitter='best'` and criterion is
+`'gini'`, `'entropy`', or `'log_loss'`, for classification' or
+`squared_error'`, `'friedman_mse'`, or `'poisson'` for regression.
+
+For each potential threshold, the splitter will evaluate the split with all the
```

Comment:
```suggestion
For each potential threshold on the non-missing data, the splitter will evaluate the split with all the
```

Just a nit to make it clear that there is no threshold applied on the missing values?